pkgs/applications/virtualization/docker/default.nix

```diff
@@ -38,6 +38,8 @@ rec {
         rev = containerdRev;
         sha256 = containerdSha256;
       };
+      nativeBuildInputs = oldAttrs.nativeBuildInputs ++ [ pkgconfig ];
+      buildInputs = oldAttrs.buildInputs ++ [ libseccomp ];
```

Comment:
Can you please add the reasoning for this change to the commit message? https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#writing-good-commit-messages

Reply:
Sure, in this case it would not compile without the additional inputs (giving errors related to each of them).

Reply:
FWIW: I asked because it wasn't obvious from dockers changelog.